pkg/injector/patch.go

```diff
@@ -19,12 +20,16 @@ import (
 func (wh *mutatingWebhook) createPatch(pod *corev1.Pod, req *v1beta1.AdmissionRequest, proxyUUID uuid.UUID) ([]byte, error) {
 	namespace := req.Namespace
 
+	// Tracks the success of the current certificate issued for envoy to connect to XDS
+	success := true
```

Comment:
Could you rename this to `certIssuanceSuccess`, otherwise in the context of the the `createPatch` function, the existing variable name can be confusing.